static/src/images/global/play-icon-gold.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@Jholder112233 is absolutely right! Latest Illustrator is playing up, so I've used just svgo (floatPrecision: 1) and removed the rest of crap by hand (including viewBox since it's identical to width/height).

play-icon-gold.svg:
```
<svg width="10" height="9" xmlns="http://www.w3.org/2000/svg"><path d="M10 4.3L.4 0 0 .3v8.4l.4.3L10 4.7v-.4z" fill="#fb0"/></svg>
```
play-icon-grey.svg:
```
<svg width="10" height="9" xmlns="http://www.w3.org/2000/svg"><path d="M10 4.3L.4 0 0 .3v8.4l.4.3L10 4.7v-.4z" fill="#DCDCDC"/></svg>
```

Czyżby Lwica z Cintry?